blocks/rich-text/index.js

```diff
@@ -796,6 +799,19 @@ export class RichText extends Component {
 		} ) );
 	}
 
+	/**
+	 * Calling onSplit means we need to abort the change done by TinyMCE.
```

Comment:
I don't really understand this function. In most places where it's used, we're also calling `preventDefault`. Shouldn't that be enough to "abort the change" that would otherwise be effected by TinyMCE ?